.github/components.yml

```diff
@@ -54,26 +54,21 @@ NPU:
 
 HETERO:
   revalidate:
-    - CPU
-    - GPU
     - HETERO
```

Comment:
I think we can remove _itself_, e.g. if HETERO has changed, it's clear that it needs to be revalidated